### ISSUE TITLE
how to start it correctly?

### ISSUE THREAD

**Issue by severak** [hammaman/SonicPi#1]
Wow! Great piece of software.

How to start it correctly?

I figured to to press Run on Buffer 0, than Run on Buffer 2 etc... But if I do this, it sounds like jump in the middle of song.

**Comment by hammaman** [comment]
Yes, you need to run each buffer in turn.  Buffer 0 sets things up, so doesn't produce any sounds.  But then you need to run each of the other buffers quickly to get everything to work in time.

**Comment by severak** [comment]
thanks a lot
